### ISSUE TITLE
[Documentation] GitLab CI docs should not use legacy `pnpm prepare`

### ISSUE THREAD

**Issue by bestickley** [pnpm/pnpm#8380]
### Contribution

- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/pnpm/pnpm/blob/main/CONTRIBUTING.md))

### Describe the user story

The [GitLab CI Docs](https://pnpm.io/continuous-integration#gitlab-ci) reference `corepack prepare` which is now legacy. I think `corepack install` is the preferred option. See [here](https://github.com/nodejs/corepack/pull/291).

### Describe the solution you'd like

N/A

### Describe the drawbacks of your solution

N/A

### Describe alternatives you've considered

N/A

**Comment by zkochan** [comment]
`corepack prepare` works on more versions of Node.js, so I don't think it is a good idea to change the docs now.

**Comment by tomasvn** [comment]
@zkochan I’ll leave my PR open, once appropriate feel free to merge it. It might seem they drop the `prepare` for `install` either in version 21 or 22.

**Comment by Shinigami92** [comment]
If this https://www.youtube.com/watch?v=I7qMwaxNNOc is true..., corepack should be slowly kicked out fully out of pnpm docs, should it?

/cc @t3dotgg

**Comment by zkochan** [comment]
They still discuss it. Removing corepack from docs will just make their decision easier.

**Comment by tomasvn** [comment]
Probably safe to close this issue, since mentions of `corepack` is removed in v12. @zkochan

**Comment by zkochan** [comment]
yes, we have removed corepack from the v11 and v12 docs. Also, v12 has all the features of corepack. v12 can even manage versions of other package managers, same as corepack: https://pnpm.io/cli/shim

```
pnpm shim add yarn
```

Then in a project:

```
pnpm add yarn
```

In existing projects the right version of yarn will be installed and executed when running yarn.
